src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -14,7 +14,13 @@ public class DirectoryInfoWrapper : DirectoryInfoBase
 
         public DirectoryInfoWrapper(DirectoryInfo directoryInfo, bool isParentPath = false)
         {
-            _directoryInfo = directoryInfo;
+            // Work around for https://github.com/dotnet/corefx/issues/616: DirectoryInfo.Exists returns 
```

Comment:
Ah.. you seem to have mentioned that. When is the coreclr fix expected to reach us?